pom.xml

```diff
@@ -146,7 +146,12 @@
 			<dependency>
 				<groupId>org.slf4j</groupId>
 				<artifactId>slf4j-api</artifactId>
-				<version>1.7.8</version>
+				<version>1.7.10</version>
+			</dependency>
+			<dependency>
+				<groupId>org.slf4j</groupId>
+				<artifactId>slf4j-simple</artifactId>
+				<version>1.7.10</version>
```

Comment:
I find setting scope in root depemdencyManagement generally a bad idea. If you use such dependency in a submodule in a different scope, then you have to declare both dependency and dependencyMangement in that submodule.

It's shouldn't be a problem in case of xchange though. Nearly all the modules have homogenic role - they don't talk to each other, and examples uses logback. Let me know if you still want me to change it.